triggers/call_notification.js

```diff
@@ -78,12 +86,18 @@ const unsubscribeHook = (z, bundle) => {
     .then(response => z.console.log(response));
 };
 
+/**
+ * Is called then a collNotification happens.
```

Comment:
when

Reply:
And `collNotification` changed to `callNotification` ;)